charts/database/values.yaml

```diff
@@ -1,13 +1,12 @@
-org: "deisci"
```

Comment:
i name spaced the values this way because these are  for the database pod and postgres details are not related to the pod